setup.cfg

```diff
@@ -1,3 +1,91 @@
+[metadata]
+name = pylint
+version = attr: pylint.__version__
+license = GPL-2.0-or-later
+license_files =
+    COPYING
+    CONTRIBUTORS.txt
+description = python code static checker
+keywords = static code analysis linter python lint
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+author = Python Code Quality Authority
+author_email = code-quality@python.org
+url = https://github.com/PyCQA/pylint
+project_urls =
+    "What's New"=https://pylint.pycqa.org/en/latest/whatsnew/
+    "Bug tracker"=https://github.com/PyCQA/pylint/issues
+    "User manual"=http://pylint.pycqa.org/en/latest/
+    "Contributing"=http://pylint.pycqa.org/en/latest/development_guide/contribute.html
+    "Technical references"=http://pylint.pycqa.org/en/latest/technical_reference/index.html
+    "Mailing list"=mailto:code-quality@python.org
+classifiers =
+    Development Status :: 6 - Mature
+    Environment :: Console
+    Intended Audience :: Developers
+    License :: OSI Approved :: GNU General Public License (GPL)
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: Implementation :: CPython
+    Programming Language :: Python :: Implementation :: PyPy
+    Topic :: Software Development :: Debuggers
+    Topic :: Software Development :: Quality Assurance
+    Topic :: Software Development :: Testing
+[options.packages.find]
+include =
+    pylint*
```

Comment:
Maybe move this section after `[options]`?
Btw, I believe you still need `packages = find:` in the `[options]` section.